maflib/schemas/gdc-2.0.0-aliquot-merged.json

```diff
@@ -3,12 +3,12 @@
 	"annotation-spec" : "gdc-2.0.0-aliquot-merged",
 	"extends" : "gdc-2.0.0-aliquot",
 	"columns" : [
-            [ "RNA_Support", "YesNoOrUnknown", "Indicates if the variant is supported (Yes) or not supported (No) by tumor RNA-Seq. If it has not been checked against RNA-Seq data, the value will be 'Unknown'." ],
+            [ "RNA_Support", "VariantSupport", "Indicates if the variant is found and alleles (Match), simply (Overlap), or is not supported (No) by tumor RNA-Seq. If it has not been checked against RNA-Seq data, the value will be 'Unknown'." ],
```

Comment:
This looks like a breaking change since it's removing the ability for this column to contain "Yes" and thus all existing MAFs will be invalid. 

I think this change warrants a new spec, version what do you think @kmhernan

Reply:
So it just depends on if this has ever been used. Even if it has been used, but everything is `Unknown` because the annotation hasn't been implemented yet (which I think is the case) then it actually would still be ok. @tzuni can confirm this. Otherwise you are right.

Reply:
None of the existing MAFS will have Yes or No in this column, the tool/wf I'm building is the first to touch it afaik

Reply:
Gotcha! This change is fine then